app/controllers/macros_controller.rb

```diff
@@ -68,10 +72,6 @@ def command_params
     params.require(:commands).permit(ids: [])
   end
 
-  def find_project
-    @project = Project.find_by_param!(params[:project_id])
-  end
-
   def find_macro
     @macro = @project.macros.find(params[:id])
```

Comment:
I didn't touch the macro. It was already there. ;)

Still, this type of things, do you guys go ahead and refactor it when you guys see something that could be improved?